ra/1.1/ra-metadata-example.xml

```diff
@@ -1,103 +1,191 @@
+     and other tools, which may, but are not required to, enforce it.
+-->
+
+<!-- These attributes indicate that each instance of this agent must specify a
+     value for the "config-file" parameter, and (since no other parameter uses
+     the same unique group) the value should be unique across all instances.
+-->
+<parameter name="config-file" unique-group="config-file" required="1">
+
+<!-- A helpful, extended description of the parameter. A user interface might
+     display it to the user when asked for help with an option. You may include
+     examples, etc.
+
+     Each parameter must have at least one longdesc, and may have multiple with
+     different "lang" attributes.
+-->
+<longdesc lang="en">
+The full path to the example daemon configuration file.
+</longdesc>
+
+<!-- A brief description of the parameter's purpose. A user interface might
+     display it to the user as a tooltip or similar.
+
+     Each parameter must have at least one shortdesc, and may have multiple with
+     different "lang" attributes.
+-->
+<shortdesc lang="en">Configuration filename</shortdesc>
+
+<!-- Allowed values, and optionally the default value. Type may be boolean,
+     string, integer, or select; with select, this must have "option" child
+     elements specifying each allowed value.
+-->
+<content type="string"/>
+
+</parameter>
+
+<!-- Since the "ip" and "port" parameters share the same unique group, the
+     combination of the two should be unique across all instances of this
+     resource type.
+
+     If required is 0 or unspecified, an instance of this resource type may
+     omit this parameter (typically, a default value will be used in that
+     case).
 -->
+<parameter name="ip" unique-group="address">
 <longdesc lang="en">
-The resource name is the directory where the filesystem will be actually
-mounted.  Please make sure it exists. 
+The local IP address the example daemon should bind to and accept requests on.
+If omitted, the wildcard address will be used.
 </longdesc>
+<shortdesc lang="en">IP address</shortdesc>
+<content type="string" default="*"/>
+</parameter>
 
-<!-- The shortdesc may be displayed by the resource manager as a tooltip or
-     equivalent -->
-<shortdesc lang="en">Mountpoint</shortdesc>
+<parameter name="port" unique-group="address">
+<longdesc lang="en">
+The port number the example daemon should accept requests on.
+</longdesc>
+<shortdesc lang="en">Port number</shortdesc>
+<content type="string" default="65535"/>
+</parameter>
 
-<!-- Further definition of the content -->
-<content type="string" default="/mnt" />
+<!-- Example of a parameter that is reloadable and uses select. Reloadable
+     implies that "mode" can be changed without requiring a restart of
+     example-daemon; perhaps the resource agent writes the mode to a file
+     that example-daemon polls periodically.
 
+     If no unique group is unspecified, different instances of this
```

Comment:
This feels weird to me. I think it should be either "If no unique group is specified" or "If unique group is unspecified".

Reply:
Whoops, you're right, that was a mistake